MainRepository/PriorityQueue.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace PQueue {

```

Comment:
```suggestion
﻿namespace PQueue;

```
Use this style for namespace. So we can avoid the curly braces.